os/selinux.md

```diff
@@ -27,11 +27,11 @@ Once you're happy that your workload is compatible with the SELinux policy,
 you can temporarily enable enforcement by running the following command as
 root:
 
-* setenforce 1
+* `setenforce 1`
```

Comment:
Let's prefix command lines in code blocks with `$`, to distinguish commands from output.

E.g., 
`$ setenforce 1`

```sh
$ cat hello.txt
Hello.
```